service/turn.rest/pom.xml

```diff
@@ -123,6 +145,68 @@
           </execution>
         </executions>
       </plugin>
+
```

Comment:
I don't see you using this docker container anywhere yet.... Is there more tests coming for this or what?  It might be difficult to test this feature end to end while having it go to coturn so I'm not sure that is required but let me know what you have in mind...